scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,85 @@
+        contract_max_size_bytes = max_contract_size)   
+        
+    compute_settings = ConfigSettingContractComputeV0(ledger_max_instructions=Int64(1000000000),
+                                                 tx_max_instructions=Int64(100000000),
+                                                 fee_rate_per_instructions_increment=Int64(100),
+                                                 tx_memory_limit=Uint32(41943040)) # 40 mb
+
+    compute_upgrade_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_COMPUTE_V0,
+        contract_compute = compute_settings)    
+    
+    contract_ledger_cost_settings = ConfigSettingContractLedgerCostV0(ledger_max_read_ledger_entries=Uint32(400),
+                                                 ledger_max_read_bytes=Uint32(2048000),
+                                                 ledger_max_write_ledger_entries=Uint32(200),
+                                                 ledger_max_write_bytes=Uint32(1024000),
+                                                 tx_max_read_ledger_entries=Uint32(30),
+                                                 tx_max_read_bytes=Uint32(133120), # 130 kb
+                                                 tx_max_write_ledger_entries=Uint32(20),
+                                                 tx_max_write_bytes=Uint32(66560), # 65 kb 
+                                                 fee_read_ledger_entry=Int64(20000),
+                                                 fee_write_ledger_entry=Int64(30000),
+                                                 fee_read1_kb=Int64(1000),
+                                                 fee_write1_kb=Int64(1000),
+                                                 bucket_list_size_bytes=Int64(32212254720), # 30 * 1024 * 1024 * 1024 for 30 GB
+                                                 bucket_list_fee_rate_low=Int64(1000),
+                                                 bucket_list_fee_rate_high=Int64(10000),
+                                                 bucket_list_growth_factor=Uint32(1))
+
+    contract_ledger_cost_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_LEDGER_COST_V0,
+        contract_ledger_cost = contract_ledger_cost_settings)        
+
+    contract_historical_data_settings = ConfigSettingContractHistoricalDataV0(fee_historical1_kb=Int64(5000))
+
+    contract_historical_data_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_HISTORICAL_DATA_V0,
+        contract_historical_data = contract_historical_data_settings)    
+
+    contract_meta_data_settings = ConfigSettingContractMetaDataV0(tx_max_extended_meta_data_size_bytes=Uint32(2048), # 2 kb
+                                                                        fee_extended_meta_data1_kb=Int64(300))  
+
+    contract_meta_data_entry = ConfigSettingEntry(
+        ConfigSettingID.CONFIG_SETTING_CONTRACT_META_DATA_V0,
```

Comment:
Interesting that this works for you... the xdr is `CONFIG_SETTING_CONTRACT_EVENTS_V0` now. Is the Python sdk not up to date? This might be an issue...